pipeline/http_post.go

```diff
@@ -0,0 +1,26 @@
+package pipeline
+
+// An HTTPPostNode will take the incoming data stream and POST it to an HTTP endpoint.
+//
+// Example:
+//    stream
+//        |window()
+//            .period(10s)
+//            .every(5s)
+//        |top('value', 10)
+//        //Post the top 10 results over the last 10s updated every 5s.
+//        |httpPost('http://example.com/api/top10')
```

Comment:
having an optional property method called `endpoint` would be useful, but I can add that once this is merged.